backend/src/api/Add.py

```diff
@@ -0,0 +1,340 @@
+from src.utils.db_utils import get_db_connection
+from src.utils.db_handler import DBHandler
+from fastapi import APIRouter, HTTPException, UploadFile, Form, File
+from src.api.BaseModel import *
+from psycopg2 import errors
+from src.api.drive_utils import *
```

Comment:
Devi usare import relativi altrimenti Render non funzia, inoltre la connessione dobbiamo aprirla ad ogni chiamata altrimenti se cade quell'unica connessione va in crash tutto il sistema. (vd. profile.py)

```suggestion
from ..utils.db_utils import get_connection, MODE
from ..utils.db_handler import DBHandler
from fastapi import APIRouter, HTTPException, UploadFile, Form, File, Depends
from .BaseModel import *
from psycopg2 import errors
from .drive_utils import *
```